sotodlib/preprocess/processes.py

```diff
@@ -31,7 +32,7 @@ class Detrend(_Preprocess):
     .. autofunction:: sotodlib.tod_ops.detrend_tod
     """
     name = "detrend"
-
+    
```

Comment:
In this file there are many many tabs like this one here. Could you please remove them? Thanks

Reply:
Done